test/catalog-compat/catcompatck/catcompatck

```diff
@@ -91,6 +91,15 @@ a  b  c
 2  1  21
 3  1  31
 1  2  12
+
+# Checks for regressions against #5802; simply having this in the catalog is
+# sufficient, so it doesn't need to be requeried in each new "epoch".
+> CREATE VIEW regression_5802 AS SELECT length(length('')::text);
```

Comment:
I think in this ancient version of Materialize, this needs to be a `MATERIALIZED` view in order to be able to query from it.